pkgs/applications/networking/cluster/k3s/builder.nix

```
@@ -331,6 +332,17 @@ let
       ldflags = versionldflags;
     }).overrideAttrs
       overrideContainerdAttrs;
+
+  # TODO (#405952): remove this patch. We had to add it to avoid a mass rebuild
+  # for the 25.05 release. Once the above PR is merged, switch back to plain util-linuxMinimal.
+  k3sUtilLinux = util-linuxMinimal.overrideAttrs (prev: {
+    patches =
+      prev.patches or [ ]
+      ++ lib.singleton (fetchpatch {
```

Nit that can be disregarded but what is the advantage of syntax over just doing:

```suggestion
      ++ [
         (fetchpatch {
```

---

At least I find this easier to read.